stdlib/3/unittest/__init__.pyi

```diff
@@ -289,7 +289,11 @@ class TestResult:
 class TextTestResult(TestResult):
```

Comment:
The source code of cpython is here.
https://github.com/python/cpython/blob/0a16bb15afec28f355bc28203b6b10610293f026/Lib/unittest/runner.py#L29-L117